core/src/main/scala/sttp/tapir/FileRange.scala

```diff
@@ -0,0 +1,7 @@
+package sttp.tapir
+
+import sttp.model.headers.Range
+
+case class FileRange(underlying: Any, range: Option[Range] = None) extends FileRangeExtension
```

Comment:
can we have `underlying: sttp.tapir.File`?

Reply:
Unfortunately, no because in some cases we still need Path

Reply:
But converting `File` to a `Path` is a matter of calling `.toPath`? Can't we assume that we always deal with `java.io.File` on the JVM, and convert if necessary?

Reply:
Unfortunately we cannot assume, in tests I found out that  for some cases in runtime the underlying is Path and tests are failing because of `ClassCastException`